app/src/main/java/com/android/sample/ui/listActivities/LikedActivities.kt

```diff
@@ -80,17 +81,23 @@ fun LikedActivitiesScreen(
           when (uiState) {
             is ListActivitiesViewModel.ActivitiesUiState.Success -> {
               if (profile == null) {
-                Text(
-                    text = "You are not logged in, Login or Register to see your liked activities",
-                    modifier =
-                        Modifier.padding(8.dp)
-                            .align(Alignment.Center)
-                            .testTag("notConnectedPrompt"),
-                    color = MaterialTheme.colorScheme.onSurface)
-                Button(
-                    onClick = { navigationActions.navigateTo(Screen.SIGN_UP) },
-                    modifier = Modifier.testTag("signInButton")) {
-                      Text("Go to Sign In Page")
+                Column(
+                    horizontalAlignment = Alignment.CenterHorizontally,
+                    verticalArrangement = Arrangement.Center,
+                    modifier = Modifier.align(Alignment.Center)) {
+                      Text(
+                          text =
+                              "You are not logged in. Login or Register to see your liked activities.",
+                          modifier = Modifier.padding(bottom = 16.dp).testTag("notConnectedPrompt"),
+                          color = MaterialTheme.colorScheme.onSurface,
+                          style = MaterialTheme.typography.bodyMedium,
+                          textAlign = TextAlign.Center)
+                      Button(
+                          onClick = { navigationActions.navigateTo(Screen.SIGN_UP) },
+                          modifier = Modifier.testTag("signInButton"),
+                      ) {
+                        Text("Go to Sign In Page", style = MaterialTheme.typography.labelLarge)
```

Comment:
Great that you use MaterialTheme for organizing the UI. I highly encourage this practice for our project !